runtime/src/assembly/api/others.ts

```diff
@@ -28,7 +31,14 @@ export function SessionKeys_generate_session_keys(data: i32, len: i32): u64 {
  * @param len i32 length (in bytes) of the arguments passed
  */
 export function TaggedTransactionQueue_validate_transaction(data: i32, len: i32): u64 {
-    return Serialiser.serialiseResult([]);
+    let input = Serialiser.deserialiseInput(data, len);
+    const size = CompactInt.fromU8a(input);
+    input = input.slice(size.encodedLength());
+    const source = input.slice(0, 1);
+    input = input.slice(1);
+    const uxt = Extrinsic.fromU8Array(input);
```

Comment:
Some magic slices are happening here maybe we can move this to some meaningful module/extract to function etc.